src/components/molecules/OakFilterDrawer/__snapshots__/OakFilterDrawer.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`OakFilterDrawer matches snapshot 1`] = `null`;
```

Comment:
This snapshot doesn't seem to be doing anything.

Reply:
snapshot now generating 👍